bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/GemObservationCB.java

```diff
@@ -99,8 +99,6 @@ public Object clone() {
         private final Option<Integer> _scienceBand;
         private final SPInstObsComp _inst;
         private final ObsExecRecord _obsExecRecord;
-        private final Set<GuideProbe> _availableGuiders;
```

Comment:
Just cleanup.  I went through the class before I started adding my changes and removed unused code and did a light cleanup in general.

 It used to be that there was logic here for instructing the seqexec when and when not to guide.  That got moved but some of the logic was apparently left behind.